### PR TITLE
docs: [BD-46] updated README file

### PR DESCRIPTION
**Pull request by PKulkoRaccoonGang**
## Description

1. Added description of CSS foundation
2. Removed SCSS foundation text block (similar to the [documentation site](https://deploy-preview-3078--paragon-openedx.netlify.app/guides/getting-started#css-foundation))
3. Changed Paragon CLI usage guide
4. Verified work Paragon CLI

## Merge Checklist

* [ ] If your update includes visual changes, have they been reviewed by a designer? Send them a link to the Netlify deploy preview, if applicable.
* [ ] Does your change adhere to the documented [style conventions](https://github.com/openedx/paragon/blob/master/docs/decisions/0012-css-styling-conventions)?
* [ ] Do any prop types have missing descriptions in the Props API tables in the documentation site (check deploy preview)?
* [ ] Were your changes tested using all available themes (see theme switcher in the header of the deploy preview, under the "Settings" icon)?
* [ ] Were your changes tested in the `example` app?
* [ ] Is there adequate test coverage for your changes?
* [ ] Consider whether this change needs to reviewed/QA'ed for accessibility (a11y). If so, please request an a11y review for the PR in the [#wg-paragon](https://openedx.slack.com/archives/C02NR285KV4) Open edX Slack channel.

## Post-merge Checklist

* [ ] Verify your changes were released to [NPM](https://www.npmjs.com/package/@openedx/paragon) at the expected version.
* [ ] If you'd like, [share](https://github.com/openedx/paragon/discussions/new?category=show-and-tell) your contribution in [#show-and-tell](https://github.com/openedx/paragon/discussions/categories/show-and-tell).
* [ ] 🎉 🙌 Celebrate! Thanks for your contribution.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @PKulkoRaccoonGang!


When this pull request is ready, tag your edX technical lead.

**Review by ihor-romaniuk — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -39,37 +39,47 @@ In your React project:
...
-**index.scss**
+To apply these theme-specific styles, include the following import:
 
 ```
-@import '~@edx/brand/paragon/fonts.scss';
-@import '~@edx/brand/paragon/variables.scss';
-@import '~@openedx/paragon/styles/scss/core/core.scss';
-@import '~@edx/brand/paragon/overrides.scss';
+@use "@my-brand/light.min.css";
 ```
```
Comment (brian-smith-tcril) [inline README.md:64]: I feel like this section is a bit ambiguous. It's not clear if someone wanting to use a theme should have

```
@use "@openedx/paragon/dist/core.min.css";
@use "@my-brand/light.min.css";
```

or just 

```
@use "@my-brand/light.min.css";
```

File: README.md [hunk 2]
```diff
@@ -39,37 +39,47 @@ In your React project:
...
+```
 
 ## Paragon CLI
 
 The Paragon CLI (Command Line Interface) is a tool that provides various utility commands to automate actions within the Open edX environment.
 
 ### Available Commands
 
-- `paragon install-theme [theme]`: Installs the specific [brand package](https://github.com/openedx/brand-openedx).
+- `paragon install-theme`: Installs the specific [brand package](https://github.com/openedx/brand-openedx).
```
Comment (brian-smith-tcril) [inline README.md:81]: With the argument `[theme]` removed the wording of "the specific brand package" feels a bit unclear. Maybe having it just be

```suggestion
- `paragon install-theme`: Installs a [brand package](https://github.com/openedx/brand-openedx).
```

would clear it up?

**Review by brian-smith-tcril — COMMENTED** [review]
Overall these changes are looking great! I left a couple comments, let me know what you think!

File: README.md (hunk 1 above)
Comment (PKulkoRaccoonGang) [reply README.md:64]: Yes, I agree
The first variant will be more understandable
Thank you!

File: README.md (hunk 2 above)
Comment (PKulkoRaccoonGang) [reply README.md:81]: Yes, corrected

**Review by brian-smith-tcril — APPROVED** [review]

**Pull request merged**